src/js/modules/RoutingApp.js

```diff
@@ -138,7 +139,7 @@ class RoutingApp extends Component {
                 path="/myaccount/sign-up"
                 key={"signUp"}
                 render={() =>
-                  session.userId ? (
+                  session ? (

```

Comment:
Session, instead of storing two keys `userId` and `headers`, is now just `headers`. To find a user based on the `headers` and without `userId`, developers must use `headers.uid === user.email ?`.